setup.py

```diff
@@ -76,9 +77,14 @@ def compute_cmake_args():
             cmake_args += ['-DCMAKE_INSTALL_PREFIX=' + _ctx.cxx_install_root]
 
         else:
-            cxxLibDir = os.path.abspath(
-                os.path.join(setuptools.__file__, "../../opentimelineio/cxx-libs")
-            )
+            if "--user" in sys.argv:
+                cxxLibDir = os.path.abspath(
+                    os.path.join(_ctx.install_usersite, "opentimelineio/cxx-libs")
+                )
+            else:
+                cxxLibDir = os.path.abspath(
+                    os.path.join(setuptools.__file__, "../../opentimelineio/cxx-libs")
+                )
```

Comment:
I know this is from the existing code, but these lines should probably be something like:
```
setuptools_parent_dir = os.path.dirname(os.path.dirname(setuptools.__file__))
cxxLibDir = os.path.abspath(os.path.join(setuptools_parent_dir, "opentimelineio", "cxx-libs"))
```

Reply:
I ended up using `distutils.sysconfig.get_python_lib()` in another commit as it seems to be the correct solution. Let me know if you prefer me to get back to using that "hack" :-)

Reply:
That distutils solution certainly does feel much better!

Reply:
Yes, that is much cleaner.  Thanks!